### ISSUE TITLE
Demo gif in README for each example with links.

### ISSUE THREAD

**Issue by pushkalkatara** [JdeRobot/VisualStates-examples#10]
Currently, only priusObstacleAvoidance behavior has the demo gif, for better visualization, create demo gif for all automatas.

**Comment by sudo-panda** [comment]
Is this fine 
[kobuki_bump_and_go](https://github.com/sudo-panda/VisualStates-examples/blob/demo-gif/kobuki_bump_and_go/kobuki_bump_and_go.gif)

**Comment by pushkalkatara** [comment]
Yes, it's great. Just a suggestion, it's better if we keep the size small and gif short. Like in kobuki bump and go example, we can show the instance where the bot is about to hit and the state changes rather than the whole behavior.

**Comment by sudo-panda** [comment]
Can we fast forward the gifs
